package.json

```diff
@@ -57,6 +57,14 @@
     "tailwindcss": "^3.0.7"
   },
   "dependencies": {
-    "open-graph-scraper": "^4.11.0"
+    "open-graph-scraper": "^4.11.0",
+    "prop-types": "^15.7.2",
+    "react": "^17.0.2",
+    "react-tooltip": "^4.2.21"
+  },
+  "eslintConfig": {
+    "globals": {
```

Comment:
put this into `.eslintrc.js` please, the point of that file is that we don't use `package.json` :D

Reply:
https://github.com/open-sauced/hot/pull/51#discussion_r777061498

We resolved this already. Check the conversation we had.

Reply:
https://github.com/open-sauced/hot/pull/51#discussion_r777021962

Reply:
@bdougie has 1337 notifications per day, he focused on the content that you should modify, in this case adding `window` to our `eslint.globals` configuration. It's still a bad practice to have that configuration in 2 places, simply put it's either in `package.json` or `.eslintrc.js` but not both. [open-sauced/open-sauced](https://github.com/open-sauced/open-sauced) is using [.eslintrc.js](https://github.com/open-sauced/open-sauced/blob/main/.eslintrc.js), would prefer we keep that convention 🍕

Reply:
Gooosh 😔